networkit/components.pyx

```diff
@@ -219,22 +278,26 @@ cdef class StronglyConnectedComponents:
 
 	def getPartition(self):
 		"""
+		getPartition()
+
 		Returns a Partition object representing the strongly connected components.
 
-		Returns:
-		--------
-		Partition
+		Returns
+		-------
+		networkit.partition
 			The strongly connected components.
 		"""
 		return Partition().setThis(self._this.getPartition())
 
 	def numberOfComponents(self):
 		"""
+		numberOfComponents()
+		
 		Returns the number of strongly connected components of the graph.
 
-		Returns:
-		--------
-		int
+		Returns
+		-------
+		count
```

Comment:
I am not sure if we should do this. In the Python docs we often mix C++ types (e.g., `NetworKit::count`) and Python types (e.g., `list`) and this is confusing. However, this issue should be addressed in another PR. For now we should probably avoid using C++ types in Python docs.

Reply:
Yes, this makes more sense than mixing both types. Also agree to defer this to another (set of) PR(s). However I will directly use Python types for previously missing doc-strings.